observations/atmosphere/aircraft.yaml.j2

```diff
@@ -7,146 +7,552 @@
+        name: ObsType/airTemperature
+      is_in: 130
+
+  # Assign the initial observation error, based on pressure (for AMDAR and MDCRS; itype=131,133)
+  - filter: Perform Action
     filter variables:
-    - name: specificHumidity
-    minvalue: 1.0E-7
-    maxvalue: 0.034999999
+    - name: airTemperature
     action:
-      name: reject
-    #
-  - filter: Bounds Check
+      name: assign error
+      error function:
+        name: ObsFunction/ObsErrorModelStepwiseLinear
+        options:
+          xvar:
+            name: MetaData/pressure
+          xvals: [100000, 95000, 90000, 85000, 80000]
+          errors: [1.4706, 1.3529, 1.2353, 1.1176, 1.0]
+    where:
+    - variable:
+        name: ObsType/airTemperature
+      is_in: 131,133
+
+  # Assign the initial observation error, based on pressure (for RECON aircraft; itype=132)
+  - filter: Perform Action
     filter variables:
-    - name: windEastward
-    - name: windNorthward
-    minvalue: -130
-    maxvalue: 130
+    - name: airTemperature
+    action:
+      name: assign error
+      error function:
+        name: ObsFunction/ObsErrorModelStepwiseLinear
+        options:
+          xvar:
+            name: MetaData/pressure
+          xvals: [110000, 100000, 95000, 90000, 85000, 35000, 30000, 25000, 20000, 15000, 10000, 7500, 5000, 4000, 3000, 2000, 1000]
+          errors: [  1.2,    1.2,   1.1,   0.9,   0.8,   0.8,   0.9,   1.2,   1.2,   1.0,   0.8,  0.8,  0.9, 0.95,  1.0, 1.25,  1.5]
+    where:
+    - variable:
+        name: ObsType/airTemperature
+      is_in: 132
+
+  # Assign the initial observation error, based on pressure (for TAMDAR itype=134)
+  - filter: Perform Action
+    filter variables:
+    - name: airTemperature
+    action:
+      name: assign error
+      error function:
+        name: ObsFunction/ObsErrorModelStepwiseLinear
+        options:
+          xvar:
+            name: MetaData/pressure
+          xvals: [100000, 95000, 90000, 85000, 80000, 60000, 40000, 7500]
+          errors: [1.5,   1.35,  1.25,  1.10,  1.0,   1.3,   1.7, 1000000000.0 ]
+    where:
+    - variable:
+        name: ObsType/airTemperature
+      is_in: 134
+
+
+  # Assign the initial observation error, based on pressure (for Canadian AMDAR itype=135)
+  - filter: Perform Action
+    filter variables:
+    - name: airTemperature
+    action:
+      name: assign error
+      error function:
+        name: ObsFunction/ObsErrorModelStepwiseLinear
+        options:
+          xvar:
+            name: MetaData/pressure
+          xvals: [100000, 95000, 90000, 85000, 80000]
+          errors: [1.4706, 1.3529, 1.2353, 1.1176, 1.0 ]
+    where:
+    - variable:
+        name: ObsType/airTemperature
+      is_in: 135
+
+  # Reject where QC = 9-15
+  - filter: Perform Action
+    filter variables:
+    - name: airTemperature 
+    where:
+    - variable: PreQC/airTemperature
+      is_in: 9-15
     action:
       name: reject
-    #
-  - filter: Bounds Check
+
+# insert errormod here
```

Comment:
Do we want to make the GSI's errormod for aircraft work in UFO?

Reply:
Tried during meeting and the results were not good. Keeping errormod out for now.